src/tests/mod.rs

```diff
@@ -3,18 +3,55 @@ use std::env::var;
 
 use crate::{
     upload::result::UploadResult::{Error, Success},
-    upload::UploadOptions,
+    upload::{resource_type::ResourceTypes, UploadOptions},
     Cloudinary, Source,
 };
 
-#[tokio::test]
-async fn test_image_upload_from_url() {
+struct CloudinaryVariable {
+    pub api_secret: String,
+    pub api_key: String,
+    pub cloud_name: String,
+}
+
+fn load_env() -> CloudinaryVariable {
     dotenv().ok();
     let api_secret = var("CLOUDINARY_API_SECRET").expect("enviroment variables not set");
     let api_key = var("CLOUDINARY_API_KEY").expect("enviroment variables not set");
     let cloud_name = var("CLOUDINARY_CLOUD_NAME").expect("enviroment variables not set");
+    CloudinaryVariable {
+        api_secret,
+        api_key,
+        cloud_name,
+    }
+}
+
+#[tokio::test]
+async fn test_raw_upload() {
+    let env_vars = load_env();
+    let cloudinary = Cloudinary::new(env_vars.api_key, env_vars.cloud_name, env_vars.api_secret);
+
+    let audio_url = "https://commons.wikimedia.org/wiki/File:Panama_National_Anthem.ogg";
```

Comment:
I would like to have a small file locally.

Reply:
Yeah that's fine, I just thought that for the test to run smoothly for everyone that clones the project, a url would work better.

Reply:
Unfortunately, URLs tend to change at the most inconvenient moment. We have https://github.com/Lurk/cloudinary_rs/tree/main/assets in the repo.